src/web/app.py

```
@@ -1146,6 +1147,109 @@ async def generate_feed_briefing(
     return {"status": "ready", "briefing": result}
 
 
+# ---------------------------------------------------------------------------
+# Briefing audio endpoints (lazy generation + streaming from DB blob)
+# ---------------------------------------------------------------------------
+
+
+def _parse_range(range_header: str, total: int) -> tuple[int, int]:
+    """Parse a Range header value into (start, end) inclusive."""
+    try:
+        unit, ranges = range_header.split("=")
```

Ruff (0.15.17)

[warning] Unpacked variable `unit` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)